libr/include/r_core.h

```diff
@@ -564,6 +564,7 @@ R_API int r_core_set_file_by_name(RBin * bin, const char *name);
 R_API void r_core_debug_rr(RCore *core, RReg *reg, int mode);
 
 /* fortune */
+R_API RList *r_core_fortune_types(void);
```

Comment:
make it internal or static inline in the header to avoid breaking the abi. add a comment for your future self to clean those apis for 5.8